server/endpoint/endpoint.go

```diff
@@ -0,0 +1,62 @@
+package endpoint
+
+import (
+	microerror "github.com/giantswarm/microkit/error"
+
+	"github.com/giantswarm/leanix-exporter/server/endpoint/exporter"
+	"github.com/giantswarm/leanix-exporter/server/endpoint/version"
+	"github.com/giantswarm/leanix-exporter/server/middleware"
+	"github.com/giantswarm/leanix-exporter/service"
+	micrologger "github.com/giantswarm/microkit/logger"
+)
+
+// Config represents the configuration used to create a endpoint.
+type Config struct {
+	// Dependencies.
+	Logger     micrologger.Logger
+	Middleware *middleware.Middleware
+	Service    *service.Service
+}
+
+// New creates a new configured endpoint.
+func New(config Config) (*Endpoint, error) {
+	var err error
+
+	var versionEndpoint *version.Endpoint
+	{
+		versionConfig := version.DefaultConfig()
+		versionConfig.Logger = config.Logger
+		versionConfig.Middleware = config.Middleware
+		versionConfig.Service = config.Service
+		versionEndpoint, err = version.New(versionConfig)
+		if err != nil {
+			return nil, microerror.MaskAny(err)
+		}
+	}
+
+	var exporterEndpoint *exporter.Endpoint
+	{
+		exporterEndpoint, err = exporter.New(exporter.Config{
+			Logger:     config.Logger,
+			Middleware: config.Middleware,
+			Service:    config.Service,
+		})
+
+		if err != nil {
+			return nil, microerror.MaskAny(err)
+		}
+	}
```

Comment:
FYI the order of blogs, definitions and properties is alphabetical if not logically grouped.